### PR TITLE
version 5.2.5

### PR DESCRIPTION
**Pull request by greeng3**
Specify library name and version:  **xz_utils/5.2.5**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/xz_utils/all/conanfile.py [hunk 1]
```diff
@@ -42,13 +42,14 @@ def configure(self):
 
     def _apply_patches(self):
         # Relax Windows SDK restriction
-        tools.replace_in_file(os.path.join(self._source_subfolder, "windows", "vs2017", "liblzma.vcxproj"),
-                              "<WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>",
-                              "<WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>")
+        if self.version < "5.2.5":
```
Comment (uilianries) [inline recipes/xz_utils/all/conanfile.py:45]:
```suggestion
        if tools.Version(self.version) < "5.2.5":
```

**Comment by uilianries** [comment]
Hi @greeng3! Thanks for your contribution, please follow the review. Also, you have to sign the [CLA](https://github.com/conan-io/conan-center-index/pull/2338#issuecomment-666541696) and ask for EAP inclusion in #4. Regards!

File: recipes/xz_utils/all/conanfile.py (hunk 1 above)
Comment (greeng3) [reply recipes/xz_utils/all/conanfile.py]: Done.

**Review by jgsogo — APPROVED** [review]

**Comment by prince-chrismc** [comment]
closed by #3119

**Pull request closed without merging**